quixstreams/app.py

```diff
@@ -61,10 +64,10 @@ class Application:
 
     What it Does:
 
-    - During user setup:
+    - On init:
         - Provides defaults or helper methods for commonly needed objects
-        - For Quix Platform Users: Configures the app for it
-            (see `Application.Quix()`)
+        - If `quix_sdk_token` is passed, configures the app for to work
+            with Quix Platform.
```

Comment:
Please use "Quix Cloud" here instead of "Platform".

We refer to it as Quix Cloud in the new docs too.

Reply:
Oh, didn't know we were doing that yet, I actually wrote cloud first and then reversed thinking we weren't doing it yet LOL